### PR TITLE
Initial change for 7.x

### PR DESCRIPTION
**Pull request by stanley-vungle**
Updating the AdMob adapter to use the 7.x API

IOS-5115
IOS-5117
IOS-5118
IOS-5119
IOS-5120
IOS-5121
IOS-5122
IOS-5205

### CODE REVIEW

**Comment by stanley-vungle** [comment]
User ID will be in a different PR because it's still pending on the rewrite PR side
https://github.com/Vungle/vng-ios-sdk/pull/143

File: adapters/Vungle/VungleAdapter/Bidding/GADMediationVungleInterstitial.m [hunk 1]
```diff
@@ -89,98 +92,69 @@ - (void)requestInterstitialAd {
...
-  if (self.isAdLoaded) {
-    // Already invoked an ad load callback.
-    return;
-  }
+- (void) interstitialAdDidFailToLoad:(VungleInterstitial *)interstitial withError:(NSError *)error {
   _adLoadCompletionHandler(nil, error);
 }
 
-- (void)willShowAd {
+- (void) interstitialAdWillPresent:(VungleInterstitial *)interstitial {
```
Comment (YueVungle) [inline adapters/Vungle/VungleAdapter/Bidding/GADMediationVungleInterstitial.m:118]: I saw we don't use `interstitial` parameter in the delegate methods. Do we still need this parameter? Thanks a lot. :)

**Review by ashinagawa — COMMENTED** [review]
Looks good for non-RTA ads.

I will come back to verify RTA ads later (Still can't get them after adding query: [PBJ-4512](https://vungle.atlassian.net/browse/PBJ-4512)

**Review by ManojBudumuru-vungle — APPROVED** [review]
LGTM.

**Review by YueVungle — APPROVED** [review]
LGTM :)

**Review by ashinagawa — APPROVED** [review]
LGTM.
- Verified ad load and play

**Pull request merged**